pkgs/misc/uboot/default.nix

```diff
@@ -145,6 +146,30 @@ in {
     ];
   };
 
+  ubootRockchipTools = stdenv.mkDerivation ({
+    src = fetchFromGitHub {
+      owner = "rockchip-linux";
+      repo = "u-boot";
+      rev = "ef1dd650042f61915c4859ecc94623a09a3529fa";
+      sha256 = "sha256-w0hp1UGxCtSBlu9YJx6KUjeHfPCB+QqoNDlLp7AshoM=";
+    };
+
+    name = "uboot-rockchip-tools";
+    configurePhase = "make allnoconfig";
+    makeFlags = "CONFIG_ARCH_ROCKCHIP=y NO_SDL=1 tools";
+    installPhase = ''
+      mkdir -p $out/bin
+      cp tools/loaderimage tools/trust_merger $out/bin
+    '';
+
+    meta = with lib; {
+      homepage = "https://opensource.rock-chips.com/wiki_U-Boot";
+      description = "Tools required to create u-boot images for rockchip based boards";
+      license = licenses.gpl2;
+      maintainers = with maintainers; [ zauberpony ];
```

Comment:
(for anyone reviewing and noticing this "discrepancy")

`maintainers/maintainer-list.nix`

```
zauberpony = {
  email = "elmar@athmer.org";
  github = "elmarx";
  githubId = 250877;
  name = "Elmar Athmer";
};
```

Reply:
yes, I renamed my github account already a few years ago. 
What do you think, should I fix/change this with a separate PR, or can I just add a commit to this PR?